tests/locator_test.go

```diff
@@ -665,3 +672,89 @@ func TestLocatorShadowDOM(t *testing.T) {
 	err = p.Click("#inner-link", common.NewFrameClickOptions(time.Second))
 	require.NoError(t, err)
 }
+
+func TestSelectOption(t *testing.T) {
+	t.Parallel()
+
+	tb := k6httpmultibin.NewHTTPMultiBin(t)
+	ts := k6tests.NewGlobalTestState(t)
+	tb.Mux.HandleFunc("/favicon.ico", func(w http.ResponseWriter, _ *http.Request) {
+		_, _ = fmt.Fprint(w, "")
+	})
+	tb.Mux.HandleFunc("/selectOption", func(w http.ResponseWriter, _ *http.Request) {
+		_, err := fmt.Fprint(w, `
+		<html>
+		<body>
+			<select name="numbers" id="numbers-options" onchange="selectOnChange(this)" multiple>
+				<option value="zero">Zero</option>
+				<option value="one">One</option>
+				<option value="two">Two</option>
+				<option value="three">Three</option>
+				<option value="four">Four</option>
+				<option value="five">Five</option>
+			</select>
+		</body>
+		</html>
+		`)
+		assert.NoError(t, err)
+	})
+
+	ts.CmdArgs = []string{
+		"k6", "run", "-",
+	}
+	ts.Stdin = bytes.NewBufferString(tb.Replacer.Replace(`
+		import { browser } from 'k6/browser';
+
+		export const options = {
+		scenarios: {
+			browser: {
+				executor: 'shared-iterations',
+				options: {
+				browser: {
+					type: 'chromium',
+				},
+				},
+			},
+		},
+		};
+
+		export default async function () {
+			const page = await browser.newPage();
+			await page.goto('HTTPBIN_IP_URL/selectOption');
+			const options = page.locator('#numbers-options');
+			await options.selectOption({label:'Five'});
+			let selectedValue = await options.inputValue();
+			if (selectedValue !== 'five') {
+				throw new Error('Expected "five" but got ' + selectedValue);
+			}
+			await options.selectOption({index:5});
+			selectedValue = await options.inputValue();
+			if (selectedValue !== 'five') {
+				throw new Error('Expected "five" but got ' + selectedValue);
+			}
+			await options.selectOption({value:'five'});
```

Comment:
TODO(olegbespalov): adjust the tests to pick different select options to check that we actually pick the intendant one, but not checking the "five" from the previous working pick